### DISCUSSION TITLE
Compat request: Tetra

### DISCUSSION THREAD

**Discussion by T3sT3ro** [vectorwing/FarmersDelight#760] [Ideas]
Hello, could you add a compatibility with Tetra mod to allow tetra's shortblades and farmer's delight knifes to be able to be recognized as tetra's short blades and for shortblades to be used on chopping boards? Or something like that, I don't feel competent to delve into details. Thanks :)

I know that there is a datapack and an addon, but I feel it would be better to have this capability natively.

[Here is some more info on integrating with tetra](https://github.com/mickelus/tetra/wiki).

**Comment by vectorwing** [comment]
Heyo! Please post suggestions in the Discussions tab next time. 👍

Tetra integration is something I've been struggling to round out for a long time... mostly due to how FD's Knives operate. I could perhaps add just the shortblade compat, if it's viable, but I can't remember if it requires the blade to be a Tetra tool or not.

We tried adding the ability to "Tetra-fy" FD knives before, but many of their features rely on tags, and Tetra tools all share the same item ID. This makes it difficult to let a Tetra knife do what FD knives do. Couldn't find a good solution for this before. 😞

**Comment by SisScarletRed** [comment]
Would it be possible to use the NBT tags Tetra uses to assign tool parts?
